ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-ref.yaml

```diff
@@ -8,6 +8,15 @@ ref:
       cpu: 1000m
       memory: 500Mi
   env:
+  - name: ARCHITECTURE
+    default: ""
+    documentation: Set the architecture for ARM image replacement
+  - name: PACKET_PLAN
```

Comment:
This isn't being used, can it be removed?

Reply:
Done